AWS/s3_to_dynamodb_lambda.py

```diff
@@ -0,0 +1,46 @@
+import os
```

Comment:
check with `pre-commit run --all-files --show-diff-on-failure` to know more about your pre commits !